### PR TITLE
bash: fix generating of equalizerrc

### PR DESCRIPTION
**Pull request by mmtj**
This PR attempts to fix generating config values in bash script.

Related issue #42

I guess that there are better way to trim leading space from variable, but this definitely works. Nevertheless I am open to suggestions to make this PR better.

### CODE REVIEW

File: bin/pulseaudio-equalizer.in [hunk 1]
```diff
@@ -74,8 +74,10 @@ if [ -f "$CONFIG_NAME" ]; then
     index=$((index+1))
     current=$((current+1))
   done
+  # trim first space
+  read  -rd '' trimmed <<< "$RAWDATA1"
```
Comment (FFY00) [inline bin/pulseaudio-equalizer.in]: There's no real clean way to trim spaces in bash, so I would like to go to the more readable option.

Can we do `trimmed=$(echo $RAWDATA1 | sed 's/ *$//g')` instead?

**Review by FFY00 — CHANGES_REQUESTED** [review]

File: bin/pulseaudio-equalizer.in (hunk 1 above)
Comment (mmtj) [reply bin/pulseaudio-equalizer.in]: I agree to go with more readable option. Sure, we can use sed.

**Review by FFY00 — APPROVED** [review]

**Comment by FFY00** [comment]
Looks good, thanks!

**Comment by FFY00** [comment]
Agh, forgot we are replacing something in the line bellow. We should have sed do that also :woman_facepalming:.

**Pull request merged**